lib/rentlinx/validators/attribute_processor_service.rb

```diff
@@ -37,7 +38,9 @@ def process
                      zip: ZipValidator,
                      leadsURL: UrlValidator,
                      city: CityValidator,
-                     address: AddressValidator }
+                     address: AddressValidator,
+                     capAmount: AmountValidator,
+                     companyCapAmount: AmountValidator }
```

Comment:
Actually...that change should be somewhere else I think. It has some repercussions.